docs/001_develop/03_client-capabilities/019_styling/01_extending-genesis-components/02_overriding-default-styles.mdx

```diff
@@ -45,15 +44,19 @@ Let's override structural properties defined in the `baseStyles` by adding or mo
 ##### Custom Base Styles
 
 ```typescript
-import { css } from '@genesislcap/web-core';
```

Comment:
question: are we moving all examples to "double quotes symbol"?

Reply:
i did format document and it must have switched it